molecule/default/molecule-with-systemd.yml

```diff
@@ -15,72 +15,80 @@ lint: |
   ansible-lint
   flake8
 platforms:
-  - name: amazonlinux2_systemd
+  - name: amazonlinux2-systemd
     image: geerlingguy/docker-amazonlinux2-ansible:latest
     privileged: yes
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     command: /lib/systemd/systemd
     pre_build_image: yes
-  - name: debian9_systemd
+  - name: debian9-systemd
     image: geerlingguy/docker-debian9-ansible:latest
     privileged: yes
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     command: /lib/systemd/systemd
     pre_build_image: yes
-  - name: debian10_systemd
+  - name: debian10-systemd
     image: geerlingguy/docker-debian10-ansible:latest
     privileged: yes
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     command: /lib/systemd/systemd
     pre_build_image: yes
-  # Debian Bullseye is not yet supported by Tanium
-  #
-  # - name: debian11_systemd
+  # We do not yet have a Debian package for Bullseye.  Presumably we
+  # could obtain one from the CDM folks.
```

Comment:
I still don't see anything past Buster on the [list of supported client platforms](https://docs.tanium.com/client/client/requirements.html#Client_host_system_requirements).

Reply:
OK.  Can we resolve these two comments or do you want me to mention that in the comments?